### PR TITLE
Add Box, Medium, OLX Polska, Overdrop, Overdrop Pro

### PR DESCRIPTION
**Pull request by KZacharski**
## Description

Fixes #(issue) 

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
✅ : Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* Box (`com.box.android`)
* Medium (`com.medium.reader`)
* OLX Polska (`pl.tablica`)
* Overdrop (`widget.dd.com.overdrop.free`)

### Icons linked
* Overdrop Pro (linked `widget.dd.com.overdrop.pro` to `@drawable/overdrop`)

### CODE REVIEW

**Review by Chefski — CHANGES_REQUESTED** [review]
Hiya! Thanks for contributing. Could you just fix the OLX icon by change the width of the Lins to be 12px. For future or this fix, I recommend just using stroke with a width of 12px as it will avoid small issues like this. Cheers

**Comment by KZacharski** [comment]
It was originally a 12px stroke but then I converted it to outlines, scaled it and forgot about it, thanks for reminding me

**Review by Chefski — APPROVED** [review]

**Comment by Chefski** [comment]
> It was originally a 12px stroke but then I converted it to outlines, scaled it and forgot about it, thanks for reminding me

No problem. Looks good now!
@Goooler

**Comment by Goooler** [comment]
@KZacharski Can you rebase? Seems I can't push to your branch.

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -483,6 +485,7 @@
     <icon drawable="@drawable/onedotonedotonedotone" package="com.cloudflare.onedotonedotonedotone" name="1.1.1.1" />
     <icon drawable="@drawable/onedrive" package="com.microsoft.skydrive" name="OneDrive" />
     <icon drawable="@drawable/onepassword" package="com.agilebits.onepassword" name="1Password" />
+    <icon drawable="@drawable/olx" package="pl.tablica" name="OLX Polska" />
```
[the remark this answers is not in the document]
Comment (Goooler) [reply app/src/main/res/xml/grayscale_icon_map.xml]: e49dfc8ed59934f6ad4aea21f37d09e1522c87bd

**Comment by Goooler** [comment]
https://github.com/LawnchairLauncher/lawnicons/actions/runs/3125960088/jobs/5070922244

CC @SuperDragonXD 

```
Traceback (most recent call last):
  File "/home/runner/work/lawnicons/lawnicons/send_notifications.py", line 55, in <module>
    commits = list(repository.iter_commits(commits_range))
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/git/objects/commit.py", line 407, in _iter_from_process_or_stream
    finalize_process(proc_or_stream)
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/git/util.py", line 386, in finalize_process
    proc.wait(**kwargs)
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/git/cmd.py", line 502, in wait
    raise GitCommandError(remove_password_if_present(self.args), status, errstr)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(128)
  cmdline: git rev-list 02ebd9423634b73514d81cf64f29e24995bf63cf...b80d2c94[20](https://github.com/LawnchairLauncher/lawnicons/actions/runs/3125960088/jobs/5070922244#step:6:21)9f3134a69177ee14036702b6ed9f4a --
  stderr: 'fatal: Invalid symmetric difference expression 02ebd94[23](https://github.com/LawnchairLauncher/lawnicons/actions/runs/3125960088/jobs/5070922244#step:6:24)634b73514d81cf64f29e[24](https://github.com/LawnchairLauncher/lawnicons/actions/runs/3125960088/jobs/5070922244#step:6:25)995bf63cf...b80d2c94209f3134a69177ee14036702b6ed9f4a
'
Error: Process completed with exit code 1.
```

**Pull request merged**